README.md

```diff
@@ -1,10 +1,11 @@
 # Ruby Project
 
+[Be a Ruby](https://www.bearuby.org/) is a local non-profit organization that distributes items that they collect through donations to people in need in the Grand Rapids community. They currently have limited inventory tracking, which makes it difficult to see what items need to be collected. Our mission is to create Ruby, a custom mobile application that will provide an intuitive interface for them to log and update their stock. To increase security and reduce errors when updating stock, the app will have administrator and site roles that limit access to reading and writing to the stock log.
+
+Currently, Be A Ruby has no organized system for keeping track of the donations they receive and distribute. While other stock tracking tools are available and used in almost every store, they’re for industrial settings where stock is marked with barcodes and usually have a high implementation cost, which isn’t practical for a small non-profit. Our app will provide Be A Ruby with a simple and customized stock tracking interface that works best for their specific needs. This will include an interface where volunteers can log any items as they remove them, and an administrator role that can receive notifications if they're running out of any items. As a stretch goal, Ruby could be further customized to their organization by include a volunteer login with volunteering information for future events and links on how to donate.
```

Comment:
Break up 2nd sentence. "where stock is marked with barcodes. This has a high implementation cost, which ..."

Reply:
administrator role: notifications for low stock items and add new items (current wording doesn't have a way to add new items)